[2024-09-10T07:13:39.573+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:13:39.603+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-12-11T00:00:00+00:00 [queued]>
[2024-09-10T07:13:39.620+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-12-11T00:00:00+00:00 [queued]>
[2024-09-10T07:13:39.621+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:13:39.675+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2022-12-11 00:00:00+00:00
[2024-09-10T07:13:39.685+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3523) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:13:39.687+0000] {standard_task_runner.py:72} INFO - Started process 3528 to run task
[2024-09-10T07:13:39.685+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2022-12-11T00:00:00+00:00', '--job-id', '504', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpy1c5n7h8']
[2024-09-10T07:13:39.688+0000] {standard_task_runner.py:105} INFO - Job 504: Subtask first_task
[2024-09-10T07:13:39.768+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2022-12-11T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:13:39.930+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-12-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-11T00:00:00+00:00'
[2024-09-10T07:13:39.933+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:13:39.956+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:13:39.957+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-10T07:13:39.971+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:13:39.974+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-10T07:13:39.975+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:13:40.004+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:13:40.005+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2022-12-11T00:00:00+00:00, execution_date=20221211T000000, start_date=20240910T071339, end_date=20240910T071340
[2024-09-10T07:13:40.070+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:13:40.095+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T07:13:40.106+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
